Content.Client/Fullscreen/FullscreenHook.cs

```diff
@@ -0,0 +1,65 @@
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using Content.Client.Viewport;
+using Content.Shared.Input;
+using Robust.Client.Graphics;
+using Robust.Client.Input;
+using Robust.Client.State;
+using Robust.Shared.ContentPack;
+using Robust.Shared.Input.Binding;
+using Robust.Shared.IoC;
+using Robust.Shared.Log;
+using Robust.Shared.Utility;
+using SixLabors.ImageSharp;
+using SixLabors.ImageSharp.PixelFormats;
+using Content.Client.UserInterface.Screens;
+using Content.Shared.CCVar;
+using Content.Shared.HUD;
+using Robust.Client.UserInterface;
+using Robust.Client.UserInterface.Controls;
+using Robust.Client.UserInterface.XAML;
+using Robust.Shared;
+using Robust.Shared.Configuration;
+using Robust.Shared.Prototypes;
+
+namespace Content.Client.Fullscreen
+{
+    internal sealed class FullscreenHook : IFullscreenHook
+    {
+        [Dependency] private readonly IInputManager _inputManager = default!;
+        [Dependency] private readonly IConfigurationManager _cfg = default!;
+
+        public void Initialize()
+        {
+            _inputManager.SetInputCommand(ContentKeyFunctions.ToggleFullscreen, InputCmdHandler.FromDelegate(_ =>
+            {
+                ToggleFullscreen();
+            }));
```

Comment:
Don't construct a lambda here, just have ToggleFullscreen take the delegate argument and discard it.